examples/bridge-app/.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Looks like this file was not intended to be committed.

Reply:
I've removed the .idea folder.